msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -193,6 +194,36 @@ public static SilentTokenCommandParameters createSilentTokenCommandParameters(
         return commandParameters;
     }
 
+    public static DeviceCodeFlowCommandParameters createDeviceCodeFlowCommandParameters(
+            @NonNull final PublicClientApplicationConfiguration configuration,
+            @NonNull final OAuth2TokenCache tokenCache,
+            @NonNull String[] scopes){
+
+        final AbstractAuthenticationScheme authenticationScheme = AuthenticationSchemeFactory.createScheme(
+                configuration.getAppContext(),
+                null
```

Comment:
What Authentication Schemes are actually supported over DCF?

Reply:
If it's just Bearer then we should hard code that here instead of `null` as the logic inside `createScheme` to handle `null` values may change and we don't want this to break when that happens

Reply:
I went ahead and hard coded a `BearerAuthenticationSchemeInternal` Object.